aea/decision_maker/base.py

```diff
@@ -301,11 +305,36 @@ def _handle_tx_message(self, tx_message: TransactionMessage) -> None:
         :param tx_message: the transaction message
         :return: None
         """
-        score_diff = self.preferences.get_score_diff_from_transaction(self.ownership_state, tx_message)
-        if score_diff >= 0.0:
-            pass  # TODO: implement tx flow
-            # envelope = Envelope()
-            # self.outbox.put(envelope)
+        amount = tx_message.get("amount")
+        m_address = self.generate_address_from_public_key(self._wallet.public_keys['fetchai'])
+        api = LedgerApi("127.0.0.1", 8100)
+        self._generate_wealth()
+        print(api.tokens.balance(m_address))
```

Comment:
You often add `print` statements. Please adapt the habit to remove them for PRs if they are only for you during development. If you want to log stuff then please use the logger :)